iceoryx_hoofs/test/moduletests/test_unit_duration.cpp

```diff
@@ -912,7 +913,7 @@ TEST(Duration_test, ConvertDaysFromDurationMoreThanOneDay)
 TEST(Duration_test, ConvertDaysFromMaxDuration)
 {
     ::testing::Test::RecordProperty("TEST_ID", "32d1bc55-d2c5-4d83-8386-d199dfffb66d");
-    constexpr uint64_t SECONDS_PER_DAY{60U * 60U * 24U};
+    constexpr uint64_t SECONDS_PER_DAY{static_cast<uint64_t>(60U * 60U * 24U)};
```

Comment:
It's somehow weird that the cast is necessary here. I'm afraid to write that, but I think you can get rid of the cast (and two of the `U`s) when you replace one `U` with `ULL` :grimacing:

Reply:
I think this wouldn't work on all platforms since sometimes an `unsigned long long` is an `uint64_t` and sometimes it isn't.